### PR TITLE
Updates from Robin

### PR DESCRIPTION
**Pull request by Robinlovelace**
- Update abstract
- Auto format with RStudio Visual Mode

### CODE REVIEW

File: README.Rmd [hunk 1]
```diff
@@ -166,29 +193,27 @@ kbl(df,
...
-
-@lee_neighbourhood_2008 found good lighting to be the most important facilitator of walking and cycling.
+Furthermore, previous work by @fotios_whole-year_2019 suggests that streetlighting can benefit bother walking and cycling.
+They found that "ambient light has a significant impact: For a given time of day, more people walk or cycle when it is daylight than after dark and more people cycle on cycle trails and walk on foot paths after dark when they are lit than when they are unlit".
+This supported previous work which "indicated the numbers of pedestrians and cyclists during the case period were significantly higher during daylight conditions than after-dark, resulting in a 62% increase in pedestrians and a 38% increase in cyclists" [@uttley_using_2017].
+Overall, these studies provided reason to take seriously a bolder finding: @lee_neighbourhood_2008 found good lighting to be the most important facilitator of walking and cycling.
+However, the relative importance of lighting, compared with other factors, has not been the subject of large empirical studies in the papers reviewed.
```
Comment (Robinlovelace) [inline README.Rmd:216]: I am not sure about this, heads-up @eugenividal and apologies if I'm reading too far into it. I want SATURN to be the first project that properly investigates this.

**Review by eugenividal — COMMENTED** [review]
Thanks for the feedback, Robin. I had a look at it and added some comments to your comments. If you do not mind, instead of merging it, I think I going to make the changes manually. I think I prefer it for these kinds of papers (mostly text writing). It gives me more control.

File: README.Rmd [hunk 2]
```diff
@@ -10,65 +10,80 @@ always_allow_html: true
...
+Previous academic literature reviews have touched on elements of the relationship between active travel and lighting.
+@cerin_neighbourhood_2017 reviewed literature on the relationship between physical environment and active travel in general; @cozens_critical_2003 explored papers on the association between street lighting and crime or fear of crime; @boyce_benefits_2019 reviewed research into the benefits of light at night for perceptions of safety; and @fotios_lighting_2017 reviewed suggested interventions to enable cycling lighting to contribute to the safety of cyclists in the UK.
 However, to the best of our knowledge, no academic paper has reviewed the international literature available on the relationship between active travel and road lighting specifically.
 
+```{=html}
```
[the remark this answers is not in the document]
Comment (Robinlovelace) [reply README.Rmd:43]: I didn't know it either, was added automatically by RStudio's Visual Editor.

**Comment by Robinlovelace** [comment]
> If you do not mind, instead of merging it, I think I going to make the changes manually. I think I prefer it for these kinds of papers (mostly text writing). It gives me more control.

I don't mind :+1:

**Comment by Robinlovelace** [comment]
Closing in favour of manual changes. Great to see this evolve.

File: README.Rmd [hunk 3]
```diff
@@ -10,65 +10,80 @@ always_allow_html: true
...
+
 This search, based on the content of the title, abstract, and keywords, resulted in 1227 records.
-After screening abstracts and full text, 56 studies were selected. We excluded papers that did not focus on active travel and road lighting. The majority of excluded papers focussed on... We excluded six papers that could not be accessed.
+After screening abstracts and full text, 56 studies were selected.
+We excluded papers that did not focus on active travel and road lighting (many studies mentioned active travel and lighting in passing but not as the topic of the paper).
+The majority of excluded papers focussed on... We excluded six papers that could not be accessed.
+<!--# Do we have a list of each, e.g. in a .csv file? -->
```
[the remark this answers is not in the document]
Comment (Robinlovelace) [reply README.Rmd:83]: Fantastic, that could be good for an appendix.

File: README.Rmd (hunk 1 above)
Comment (Robinlovelace) [reply README.Rmd:216]: > That's actually the point of doing it, right?

Yes :+1:

**Comment by eugenividal** [comment]
Great! Thanks, Robin.

**Comment by eugenividal** [comment]
Hi @Robinlovelace. This morning I woke up at 5 and I thought that maybe one way to improve the review would be to focus only on the first topic, that is, on **active travel (AT) behavior and lighting**. As you saw yesterday, with the search I identified 5 different themes (1. AT behaviour & lighting; 2. AT, road safety & lighting; 3. AT, crime & lighting; 4. AT inequalities & lighting; and 5. AT and proper lighting). But this doesn't mean we have to focus on all of them in the review. We may need to mention all of them, as these are related topics, but not in the same detail. In that case, there would be only one Research Question: to what extent does darkness/road lighting influence active travel behaviour? and the title of the review would be 'Active Travel Behavior (or uptake) and Road Lighting: A Review. 

Perhaps, the best, for the time being, will be to progress on the AT behaviour & lighting theme results section now, which is the first one, and depending on how long and interesting looks, we can decide if carry on with the rest of the themes with the same detail or focus just on this. What do you think?

**Comment by Robinlovelace** [comment]
Sounds reasonable to me, narrowing the scope seems reasonable. Yes, we can outline all the the themes but focus on 1, a good strategy as the paper starts broad and 'zooms in' to the more specific topic. Great thinking Eugeni!

**Comment by eugenividal** [comment]
Well, let's see how evolves. Before, I meant I woke up at 5, had the idea, but then I came back to sleep :)

**Pull request closed without merging**